xml/cl.xml

```diff
@@ -7030,5 +7030,10 @@ server's OpenCL/api-docs repository.
                 <command name="clGetImageRequirementsInfoEXT"/>
             </require>
         </extension>
+        <extension name="cl_intel_queue_no_sync_operations" supported="opencl">
+            <require comment="cl_command_queue_properties">
+                <enum name="CL_QUEUE_NO_SYNC_OPERATIONS_INTEL"/>
```

Comment:
We'll need to define the value of this enum in the [queue properties enums](https://github.com/KhronosGroup/OpenCL-Docs/blob/a19ed015a9fbc3b744a89a59afd00a3afe03a761/xml/cl.xml#L793) block too
```xml
 <enum bitops="27"            name="CL_QUEUE_NO_SYNC_OPERATIONS_INTEL">
```